nixos/modules/services/networking/dhcpcd.nix

```diff
@@ -207,13 +207,19 @@ in
 
         serviceConfig =
           { Type = "forking";
-            PIDFile = "/run/dhcpcd.pid";
+            PIDFile = "/run/dhcpcd/pid";
             ExecStart = "@${dhcpcd}/sbin/dhcpcd dhcpcd --quiet ${optionalString cfg.persistent "--persistent"} --config ${dhcpcdConf}";
             ExecReload = "${dhcpcd}/sbin/dhcpcd --rebind";
             Restart = "always";
           };
       };
 
+    users.users._dhcpcd = {
+      isSystemUser = true;
+      group = "_dhcpcd";
+    };
+    users.groups._dhcpcd = {};
```

Comment:
Why the leading underscore for the user and the group?

Reply:
that's just the default dhcpcd uses. we have to explicitly set it in the derivation anyway though, so anything would be fine.

Reply:
Okay, thanks! I just looked at Arch and Gentoo - both are using `dhcpcd` as user. And since we don't generally use underscores in usernames, we shouldn't follow the weird default. Pushed a commit for the rename.

Reply:
applied here, looks all good :+1: